### DISCUSSION TITLE
Can not install SEB_3.3.1.388

### DISCUSSION THREAD

**Discussion by BernardoCunha** [SafeExamBrowser/seb-win-refactoring#236] [Q&A] [answered]
I have been trying to install SEB_3.3.1.388 using "SEB_3.3.1.388_SetupBundle.exe" and always get an 0x80070490 error.
I already manually installed the following dependencies as you suggested in the #140 ticket:

    .NET Framework 4.7.2 Runtime: https://dotnet.microsoft.com/download/dotnet-framework/net472 
(I've also tried .NET Framework 5.0 Runtime - the most current one)
    Microsoft Edge WebView2 Runtime: https://go.microsoft.com/fwlink/p/?LinkId=2124703
    Visual C++ 2015-2019 Redistributable: https://support.microsoft.com/en-us/help/2977003/the-latest-supported-visual-c-downloads

I also tried to install the previous "SEB 3.2.0.324 (x64): https://sebdev-let.ethz.ch/api/buildjobs/tgvq4q38nexl26m0/artifacts/SEB_3.2.0.324_x64_Setup.msi" as suggested in that ticket and **it installs correctly and runs OK.**

The log file and error screen follows. Can you suggest me what to do for getting SEB_3.3.1.388 working?
Thanks in advance

[Safe_Exam_Browser_20211103170058.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/7474694/Safe_Exam_Browser_20211103170058.log)
![SEB_3_3](https://user-images.githubusercontent.com/17931519/140312403-166cb77d-b9f5-4975-8790-9e717707fbe6.jpg)

**Comment by BernardoCunha** [comment, marked as answer]
Sorry. I have just tried to install from the "SEB_3.3.1.388_x64_Setup.msi" (instead of the "_SetupBundle" ), after all the dependencies were manually installed, and it worked OK.

Thanks anyway

**Reply by dbuechel** [reply]
Excellent, glad to know that.

**Comment by dbuechel** [comment]
According to the log, it appears that the installation of the Visual C++ 2015-2019 Redistributable (specifically `vc_redist.x64.exe`) has failed. Looking at the window frame as well as the OS info in the log you posted, you appear to be using Windows 7; please note that SEB 3.x does not officially support Windows 7 anymore. It may be that this is the root cause of your problem.

**Comment by smasaae** [comment]
:warning: **IMPORTANT** :warning:
**We clearly do not recommend average users to manipulate their certificate store!** Do only follow these steps if you know exactly what you're doing. If you're still using Windows 7 or 8, we highly recommend updating to Windows 10 or 11.

---
Original message:

look at my post for win 10, for win 7 (min. SP 1) do install ms edge first, then Visual C++ 2015-2019, add MicRooCerAut2011_2011_03_22.crt to local computer certificate, do install .NET 4.7.2, then install SEB_3.3.1.388_SetupBundle.exe with internet connection online, good luck

**Reply by dbuechel** [reply]
@smasaae Could you please elaborate on what kind of certificate "MicRooCerAut2011_2011_03_22.crt" exactly is and why users would need to install it on their system?

SEB 3.x does not support Windows 7 anymore (neither does MSFT themselves, at least officially). If your instructions allow to use SEB 3.x under Windows 7 and you know exactly what that certificate does, then great. If it is some random hack which "makes things work", then I'm afraid I will need to delete your comment as it could endanger the system security and/or integrity of other users which might not be technically skilled enough to realize the dangers which importing a certificate can bring.

**Reply by smasaae** [reply]
its necesarry for installing .NET 4.7.2 for win 7, to counter error Timestamp signature or certificate could not be verified or malformed, tho firstly u must install for KB3033929 
Windows 7 64 Bit- https://www.microsoft.com/en-us/download/details.aspx?id=46148
Windows 7 32 Bit- https://www.microsoft.com/en-ie/download/details.aspx?id=46078
and then add those certificate berfore installing .NET 4.7.2

**Reply by dbuechel** [reply]
Okay, thanks for the context. I think I've found the official page from Microsoft providing the "Microsoft Root Certificate Authority 2011": https://www.microsoft.com/pkiops/Docs/Repository.htm.
